.sync.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
from https://github.com/voxpupuli/plumbing/blob/master/share/travis_secrets.yaml#L15:

```yaml
.travis.yml:
  secure: "hFYhJJheFPnsggQpLK/hMSc6/JIWo9Y8v+trpYAkbAWpe+bHd/O507fHsjuOAWJAEMxkeUHdT1/YqKVQ7f8WWs8Avv3RSeA5prK645tGBwjCLbP1LVb7efDM793FMrSfaFlOM3qp5JlXPTS4CH1+5ULqzZ5ikoxxS0FoUSD8FBDxA50EsrFYgkah5Ch6BeKJAzbG3EInLskGzZ0ra1B6s9yQwBB7S/yH7XoBL2gbTbXrP/EnlsbFxLTBYInsuk5Pfsz0OEC4wFkSRegQ/q9fKye2MncEOf+Gb2QvxZMBRMRn1r/um7LpLdfOFx+tXDttS5qigRyFX8b11Vwi4MuKqIU7DcAFA7LkjWEVXo0DcM9HAeL6z4X4GWgHW3ihZvKTGYdg7QGdL8n6NGEk6cEGub1UotyVfvLEx6rwpckdrFyqrblmCNuiI1PMv2KN5Kq3fmuC0DjyHaw8OYZGtJEzk3ZVSm33ZaimUh750ck985u2v1PyoNo2GReDVyFIek7qas5X6cV4ypwCKpzFGjX9ZRV2wvzWHMQ2NztAka+BfiqCNtTVj3foSYnWPNp/9Ixlr1jETQuw15YktaGs5yOOG8MHouwS6xzWKH1jfrEJktEQ/kO0u/zwjMl+7DlwZEUzyD5+ElxaczgKNE3pZNxCL9ZkhjVRgiE+jmbRnwSsUDg="
```

Reply:
I know, but this will not fix all these errors. :/